### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by stamphamster** [lllyasviel/Fooocus#3361] [Q&A] [answered]
Did anyone manage to run the API? If yes, do you guys have an example code?

**Comment by mashb1t** [comment, marked as answer]
You can find an example for 2.4.X here: https://github.com/toutjavascript/FoooXus-Fooocus-Extender/blob/main/src/api.py#L223

Keep in mind to adjust it for the new enhance attributes.

**Reply by stamphamster** [reply]
Thanks a lot! Ill give it a go. 

Highly appreciate your feedback!

**Comment by stamphamster** [comment]
I could not make it work with that one, but was successful with https://github.com/mrhan1993/Fooocus-API

**Reply by mashb1t** [reply]
@stamphamster just keep in mind that the REST API does not support real-time updates using websockets.
